cinema/urls.py

```diff
@@ -1,10 +1,28 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+from cinema.views import (
+    movie_list,
+    movie_detail,
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    CinemaHallViewSet,
+    MovieViewSet,
+)
 
-from cinema.views import movie_list, movie_detail
+router = DefaultRouter()
+router.register(r"cinema_halls", CinemaHallViewSet, basename="cinema_hall")
+router.register(r"movies", MovieViewSet)
+
+app_name = "cinema"
 
 urlpatterns = [
     path("movies/", movie_list, name="movie-list"),
     path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("genres/", GenreList.as_view(), name="genre-list"),
+    path("genres/<int:pk>/", GenreDetail.as_view(), name="genre-detail"),
+    path("actors/", ActorList.as_view(), name="actor-list"),
+    path("actors/<int:pk>/", ActorDetail.as_view(), name="actor-detail"),
+    path("", include(router.urls)),
```

Comment:
you have movies at the top of the list and now to connect router with "movies" url and viewset. Don't you think it'll conflict?